netutils/utils.py

```diff
@@ -0,0 +1,81 @@
+"""Utilities for the netutils library."""
+from importlib import import_module
+
+_JINJA2_FUNCTION_MAPPINGS = {
+    "asn_to_int": "asn.asn_to_int",
+    "config": {
```

Comment:
I don't understand this nesting

Reply:
I definitely made this more complicated than it needed to be. I blame it being Sunday when I did it and not know about `rstrip()`